api/client/inspect.go

```diff
@@ -57,20 +61,33 @@ func (cli *DockerCli) CmdInspect(args ...string) error {
 				continue
 			}
 		} else {
-			var value interface{}
-
 			// Do not use `json.Unmarshal()` because unmarshal JSON into
 			// an interface value, Unmarshal stores JSON numbers in
 			// float64, which is different from `json.Indent()` does.
```

Comment:
Yep that test works: DockerSuite.TestInspectInt64                                0.76
I am changing the code.